driver-async/src/test/functional/com/mongodb/async/client/ClientSideEncryptionExternalKeyVaultTest.java

```diff
@@ -117,16 +103,14 @@ public void setUp() throws IOException, URISyntaxException {
         MongoClientSettings externalClientSettings = null;
         if (withExternalKeyVault) {
             externalClientSettings = clientSettingsBuilder
-                    .codecRegistry(codecRegistry)
                     .credential(MongoCredential.createCredential("fake-user", "admin", "fake-pwd".toCharArray()))
                     .build();
             autoEncryptionSettingsBuilder.keyVaultMongoClientSettings(externalClientSettings);
         }
 
         AutoEncryptionSettings autoEncryptionSettings = autoEncryptionSettingsBuilder.build();
 
-        clientSettings = clientSettingsBuilder
-                .codecRegistry(codecRegistry)
+        MongoClientSettings clientSettings = clientSettingsBuilder
```

Comment:
The builder is mutable.  So by modifying it here (adding autoEncryptionSettings) that modification remains when it's reused below.  I suggest just getting a new one for each usage.

Reply:
done

Reply:
done